service/src/utils/node-event-listeners.ts

```diff
@@ -3,7 +3,12 @@ import { archLogger } from "./chalk-theme";
 
 const idTruncateLimit = 5;
 
-const discoverPeers: string[] = [];
+// <nodeName>:<peerId>[]
+const discoveredPeers: string[] = [];
+
+function formatDP(name, peerId) {
+    return `${name}:${peerId}`
+}
```

Comment:
To handle logging for multiple nodes, the peer list needed to be unique for name/peerId (rather than just peerId)